### PR TITLE
Lesson14 fercan

### PR DESCRIPTION
**Pull request by fers9021**

### CODE REVIEW

**Comment by fers9021** [comment]
last one

File: Lesson14/fercan/script.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// 9. Declare a variable number6 and initialize it to 6. Declare a variable number7 and initialize it to `++number6` (number6 incremented using prexif ++ operator). Log both values to the console.
+let number6 = 6 ;
+number7 = '++number6' ;
```
Comment (annasvst) [inline Lesson14/fercan/script.js]: You need to declare a variable number7. Proper way of doing it is to use either 'let' or 'const' keyword in the name of the variable.
Sice we want to set number7 to number6 + 1, don't use quotes around `++number6`.
```suggestion
const number7 = ++number6;
```

File: Lesson14/fercan/script.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+   
+
+console.log('Exercise 9:, ${number7}');
```
Comment (annasvst) [inline Lesson14/fercan/script.js]:
```suggestion
console.log('Exercise 9:, ${number6}, ${number7}');
```

File: Lesson14/fercan/script.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+console.log('------------------------');
+
+
+// 10. Declare a variable number8 and initialize it to 8. Declare a variable number9 and initialize it to `number8++` (number8 incremented using postfix ++ operator). Log both values to the console.
+let number8 = 8;
+number9 = 'number8++';
```
Comment (annasvst) [inline Lesson14/fercan/script.js]:
```suggestion
const number9 = number8++;
```

File: Lesson14/fercan/script.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+console.log('number8, number9');
+
+
+// 11. Declare a variable number10 and initialize it to -3. Use the postfix ++ operator to increment the value of num by 1, then multiply the result by 2 and log it to the console.
+let number10 = -3;
+number10 = (number10++)*2;
+console.log('number10');
```
Comment (annasvst) [inline Lesson14/fercan/script.js]: This console.log does not log the value of `number10` variable but only logs a string 'number10' (which is not really related to the `number10` variable).
```suggestion
console.log('Ex. 11: number10 is', number10);
```

File: Lesson14/fercan/script.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+
+// 12. Declare two variables a and b and initialize them to 2 and 3, respectively. Use the prefix ++ operator to increment the value of a by 1, then add b to the result and log it to the console.
+let a= 2;
+let b= 3;
+let newResult = (++a)+b;
+
+console.log('newResult');
```
Comment (annasvst) [inline Lesson14/fercan/script.js]:
```suggestion
console.log('Ex.2 newResult:', newResult);  
```

**Review by annasvst — CHANGES_REQUESTED** [review]

**Review by fers9021 — COMMENTED** [review]
ok

**Review by fers9021 — COMMENTED** [review]
ok

**Review by fers9021 — COMMENTED** [review]
ok

**Review by fers9021 — COMMENTED** [review]
ok

**Review by fers9021 — COMMENTED** [review]
ok

**Review by fers9021 — COMMENTED** [review]
ok

**Review by fers9021 — COMMENTED** [review]
ok

**Review by fers9021 — COMMENTED** [review]
ok

**Review by annasvst — APPROVED** [review]
Looks good, approved ✅

**Pull request merged**